Pillow introduced pybind11 to improve parallel compilation.
We can safely remove it to get the same results.

https://github.com/python-pillow/Pillow/issues/8971
https://github.com/python-pillow/Pillow/commit/2059e060051acd2024360834da435a266d9dc665

--- pillow-13.0.0/pyproject.toml
+++ pillow-13.0.0/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 build-backend = "backend"
 requires = [
-  "pybind11",
   "setuptools>=77",
 ]
 backend-path = [
--- pillow-13.0.0/setup.py
+++ pillow-13.0.0/setup.py
@@ -17,7 +17,6 @@ import sys
 import sysconfig
 import warnings
 
-from pybind11.setup_helpers import ParallelCompile
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
@@ -35,7 +34,6 @@ while sys.argv[-1].startswith("--pillow-
     configuration.setdefault(key, []).append(value)
 
 parallel_default = int(configuration.get("parallel", ["0"])[-1])
-ParallelCompile("MAX_CONCURRENCY", parallel_default).install()
 
 
 def get_version() -> str:
